source/ui/BioXAS/BioXASMainXASScanConfigurationView.cpp

```diff
@@ -232,3 +241,14 @@ void BioXASMainXASScanConfigurationView::onEdgeChanged()
 	if (energy_->value() != configuration_->energy())
 		energy_->setValue(configuration_->energy());
 }
+
+void BioXASMainXASScanConfigurationView::onEstimatedTimeChanged()
+{
+    configuration_->blockSignals(true);
```

Comment:
I saw thats how REIXS had it set up. My impression was that it was required to get the information from the config.